### PR TITLE
Getting started document for scalar db on dynamo db

### PR DESCRIPTION
**Pull request by scalar-boney**
Getting started document for scalar db on dynamo db

Task: https://scalar-labs.atlassian.net/browse/DLT-7541

Note:- The current master branch does not support Dynamo DB.

### CODE REVIEW

File: docs/getting-started-with-dynamodb.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Scalar DB is written in Java. So the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+        
+From here, we assume Oracle JDK 8 is properly installed in your local environment.
+
+## Configure Scalar DB
+    
+The **scalardb.properties** (getting-started/scalardb.properties) file holds the configuration for Scalar DB. You need to update `contact_points` with aws region, `username` with your aws access key id, `password` with your aws access secret key and `storage` with `dynamo`.
```
Comment (feeblefakie) [inline docs/getting-started-with-dynamodb.md:17]:
```suggestion
The **scalardb.properties** (getting-started/scalardb.properties) file holds the configuration for Scalar DB. You need to update `contact_points` with AWS region, `username` with your AWS access key id, `password` with your AWS access secret key and `storage` with `dynamo`.
```

File: docs/getting-started-with-dynamodb.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Getting Started with Scalar DB on Dynamo DB
+
+## Overview
+This document briefly explains how you can get started with Scalar DB on Dynamo DB with a simple electronic money application.
```
Comment (feeblefakie) [inline docs/getting-started-with-dynamodb.md:4]:
```suggestion
This document briefly explains how you can get started with Scalar DB on DynamoDB with a simple electronic money application.
```

File: docs/README.md [hunk 3]
```diff
@@ -6,3 +6,4 @@ This document briefly explains how you can get started with Scalar DB with a sim
 ## Getting Started
 ### [Electronic Money Application with Scalar DB on Cassandra](getting-started-with-cassandra.md)
 ### [Electronic Money Application with Scalar DB on Cosmos DB](getting-started-with-cosmosdb.md)
+### [Electronic Money Application with Scalar DB on Dynamo DB](getting-started-with-dynamodb.md)
```
Comment (feeblefakie) [inline docs/README.md:9]:
```suggestion
### [Electronic Money Application with Scalar DB on DynamoDB](getting-started-with-dynamodb.md)
```

**Review by feeblefakie — CHANGES_REQUESTED** [review]
You tend to use improper naming. (not rigorous enough)
Please check such things by yourself before creating a PR so that we can reduce the number of interactions.

As you can see, the correct name is `DynamoDB` not `Dynamo DB`.
On the other hand, `Cosmos DB` is correct and `CosmosDB` is wrong.
Likewise, don't use `aws`. It's `AWS`.

Please don't make us point out such minor things every time.

File: docs/getting-started-with-dynamodb.md (hunk 1 above)
Comment (scalar-boney) [reply docs/getting-started-with-dynamodb.md]: Thank you for your suggestions.
Sorry for the inconvenience, I will check this kind of mistakes before creating a PR.
Document updated accordingly.

File: docs/getting-started-with-dynamodb.md (hunk 2 above)
Comment (scalar-boney) [reply docs/getting-started-with-dynamodb.md]: Thank you for your suggestions.
Document updated accordingly.

File: docs/README.md (hunk 3 above)
Comment (scalar-boney) [reply docs/README.md]: Thank you for your suggestions.
Document updated accordingly.

**Review by feeblefakie — APPROVED** [review]
LGTM.

**Review by yito88 — APPROVED** [review]
LGTM

**Pull request merged**